core/src/main/scala/za/co/absa/fadb/status/aggregation/implementations/ByMajorityErrorsStatusAggregator.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.fadb.status.aggregation.implementations
+
+import za.co.absa.fadb.status.aggregation.StatusAggregator
+import za.co.absa.fadb.status.{ExceptionOrStatusWithDataResultAgg, ExceptionOrStatusWithDataRow}
+
+/**
+  *  `ByMajorityErrorsStatusAggregator` is a trait that extends the `StatusAggregator` interface.
+  *  It provides an implementation for aggregating error statuses of a function invocation into a single error
+  *  by choosing the error that occurred the most.
+  */
+trait ByMajorityErrorsStatusAggregator extends StatusAggregator {
```

Comment:
I don't know if this is needed. I can hardly imagine someone would want to aggregate errors in this way.